server/restful/restfultest/restfultest.go

```diff
@@ -45,20 +49,46 @@ func New(schema interface{}) (*Container, error) {
 	}
 	schemaType := reflect.TypeOf(schema)
 	schemaValue := reflect.ValueOf(schema)
+	var schemaName string
+	tokens := strings.Split(schemaType.String(), ".")
+	if len(tokens) >= 1 {
+		schemaName = tokens[len(tokens)-1]
+	}
 	for _, route := range routes {
 		method, exist := schemaType.MethodByName(route.ResourceFuncName)
 		if !exist {
 			openlogging.GetLogger().Errorf("router func can not find: %s", route.ResourceFuncName)
 			return nil, fmt.Errorf("router func can not find: %s", route.ResourceFuncName)
 		}
+
 		handler := func(req *restful.Request, rep *restful.Response) {
+			inv, err := chassisRestful.HTTPRequest2Invocation(req, schemaName, method.Name)
+			if err != nil {
+				openlogging.GetLogger().Errorf("transfer http request to invocation failed, err [%s]", err.Error())
```

Comment:
all of log must use openlogging.WithTag to record info

search “openlogging.WithTag”, you will find example in this project